doc/classes/GraphEdit.xml

```diff
@@ -344,13 +344,14 @@
 		</signal>
 		<signal name="copy_nodes_request">
 			<description>
-				Emitted when the user presses [kbd]Ctrl + C[/kbd].
+				Emitted when this [GraphEdit] requests [GraphElement] nodes to be copied, for example by pressing [kbd]Ctrl + C[/kbd] (the [code]ui_copy[/code] action).
 			</description>
 		</signal>
 		<signal name="delete_nodes_request">
 			<param index="0" name="nodes" type="StringName[]" />
 			<description>
-				Emitted when attempting to remove a GraphNode from the GraphEdit. Provides a list of node names to be removed (all selected nodes, excluding nodes without closing button).
+				Emitted when this [GraphEdit] requests [GraphNode] nodes to be deleted, for example by pressing [kbd]Delete[/kbd] (the [code]ui_graph_delete[/code] action).
```

Comment:
Same as above, but without the second sentence (it's handled by the following paragraph).